charts/core/src/Chart/hooks/updateUtils.spec.ts

```diff
@@ -14,6 +14,17 @@ describe('@lg-charts/core/Chart/hooks/updateUtils', () => {
     expect(updatedOptions.series?.[1].name).toBe(newSeriesName);
   });
 
+  test('should not add a series if a chart with the same name exists', () => {
```

Comment:
nit: add the name of the util we're testing to the test name (i.e. "`addSeries` should ..."